paper/src/main/resources/config.yml

```diff
@@ -428,38 +428,74 @@ hacks:
         - "We extend our appreciation to the devs, admins, and players at: &oreddit.com/r/Civcraft"
   InvControl:
     enabled: true
+
+  Limits:
+    enabled: false
+    chunk_limits:
+      cache_configuration:
```

Comment:
Can you explain a little bit what this cache is for?

Reply:
Documentation is for the weak.

Reply:
Do we have documentation? :)